src/util.h

```diff
@@ -11,11 +11,21 @@
 #include "libsecp256k1-config.h"
 #endif
 
-#include <stdlib.h>
 #include <stdint.h>
-#include <stdio.h>
 #include <limits.h>
 
+#ifndef USE_EXTERNAL_DEFAULT_CALLBACKS
+#include <stdio.h>
+#endif
+
+#ifdef VERIFY
+#include <stdio.h>
+#endif
```

Comment:
maybe
```
#if defined(VERIFY) && !defined(USE_EXTERNAL_DEFAULT_CALLBACKS)
#include <stdio.h>
#endif
```

Reply:
Cheers, rebased and used the suggestion in patch 1.

Can a maintainer please confirm you guys use this process, that this rebasing to keep the PR consisting of only the valid patch series.

Reply:
Yes, that's the process.

Reply:
Thanks mate.